### ISSUE TITLE
Chrome always have the Experiments icon in the toolbar

### ISSUE THREAD

**Issue by djfjeff** [flathub/com.google.Chrome#258]
After installation with flatpak on Fedora 38, the Chrome UI always have the Experiments icon in the toolbar. However, this should not be the case because it is not running Chrome Beta or Dev...

```
flatpak list --app

Google Chrome                                     com.google.Chrome                                120.0.6099.109-1             stable                  system
```

![image](https://github.com/flathub/com.google.Chrome/assets/10726912/ae8f7774-59b0-448f-ba49-7d3e494b2710)

**Comment by Bohne13** [comment]
this is not flatpak related. The experimental icon is there as default since chrome 119 I think.
So you can use – as stable user – turn on or of some selected beta features.

**Comment by djfjeff** [comment]
This is not the case, both my Chrome 119 on Linux (normal RPM install) and Chrome OS does not have this icon enabled and I have default flags in all cases.

**Comment by djfjeff** [comment]
I just did some testing on this. I installed the latest flatpak on Silverblue 39 and it is showing the Experiments icon in the toolbar by default.

```
Google Chrome                       com.google.Chrome                      120.0.6099.216-1           stable         flathub         system
```

I then spined up a Fedora Workstation 39 VM and installed Chrome from rpm from the Google repo and it did not show the icon in the toolbar.

```
google-chrome-stable-120.0.6099.216-1.x86_64
```

None of the browser were signed in an account, so it is all default behavior.

So it seems that the flatpak version is detecting a development or beta Chrome version and is showing this icon
